.github/workflows/update-github.yaml

```diff
@@ -0,0 +1,45 @@
+name: "UpdateOctokit"
+
+on:
+  schedule:
+    - cron: '0 18 * * 0'
+
+jobs:
+  UpdateOctokit:
```

Comment:
will forks inherit this cron workflow? Should we add a job-condition so it doesnt run on forks?